csrc/dispatch_utils.h

```diff
@@ -21,6 +21,11 @@
 #define VLLM_DISPATCH_FLOATING_TYPES(TYPE, NAME, ...) \
   AT_DISPATCH_SWITCH(TYPE, NAME, VLLM_DISPATCH_CASE_FLOATING_TYPES(__VA_ARGS__))
 
+#define VLLM_DISPATCH_FLOATING_TYPES_AND2(SCALARTYPE1, SCALARTYPE2, TYPE, \
```

Comment:
what's this for? can you add some comments here?

Reply:
just copy from ipex and rename IPEX_DISPATCH_FLOATING_TYPES_AND2 to VLLM__DISPATCH_FLOATING_TYPES_AND2.

see :
#define IPEX_DISPATCH_FLOATING_TYPES_AND2(     \
    SCALARTYPE1, SCALARTYPE2, TYPE, NAME, ...) \
  AT_DISPATCH_FLOATING_TYPES_AND2(             \
      SCALARTYPE1, SCALARTYPE2, TYPE, NAME, __VA_ARGS__)

may be we can use AT_DISPATCH_FLOATING_TYPES_AND2 directly ?

Reply:
lora op only use limited types bf16 and half , float is not supported yet.

Reply:
please check this https://github.com/vllm-project/vllm/blob/main/csrc/dispatch_utils.h#L22-L24

Reply:
Ok, let's follow vllm , but still confuse to me .